test/utils/answersHelpers/consignment/houseConsignment/ConsignmentItemAnswersHelperSpec.scala

```diff
@@ -111,4 +114,161 @@ class ConsignmentItemAnswersHelperSpec extends AnswersHelperSpecBase {
       }
     }
   }
+
+  "cusCodeRow" - {
+
+    val cusCode = Gen.alphaLowerStr.sample.value
+
+    "must return None" - {
+      s"when $CustomsUnionAndStatisticsCodePage undefined" in {
+
+        val helper = new ConsignmentItemAnswersHelper(emptyUserAnswers, hcIndex, itemIndex)
+        val result = helper.cusCodeRow(index, itemIndex)
+        result mustBe None
+      }
+    }
+
+    "must return Some(Row)" - {
+      s"when $CustomsUnionAndStatisticsCodePage defined" in {
+
+        val json: JsObject = Json
+          .parse(s"""
+                    | {
+                    |    "Consignment" : {
+                    |      "HouseConsignment" : [
+                    |        {
+                    |          "ConsignmentItem" : [
+                    |              {
+                    |                  "Commodity" : {
+                    |                      "cusCode" : "$cusCode"
+                    |                  }
+                    |              }
+                    |          ]
+                    |        }
+                    |      ]
+                    |    }
+                    |}
```

Comment:
Can we set the values in user answers instead? Defining the answers like this makes these tests very dependent on the paths in the page objects